docker-compose.yml

```diff
@@ -33,8 +33,17 @@ services:
       REDASH_LOG_LEVEL: "INFO"
       REDASH_REDIS_URL: "redis://redis:6379/0"
       REDASH_DATABASE_URL: "postgresql://postgres@postgres/postgres"
-      QUEUES: "queries,scheduled_queries,celery"
-      WORKERS_COUNT: 2
+  generalrq:
+    build:
+      context: .
+      dockerfile: Dockerfile.generalrq
+    command: worker
+    environment:
+      PYTHONUNBUFFERED: 0
+      REDASH_LOG_LEVEL: "INFO"
+      REDASH_REDIS_URL: "redis://redis:6379/0"
+      REDASH_DATABASE_URL: "postgresql://postgres@postgres/postgres"
+      QUEUES: "periodic emails default"
```

Comment:
Do we no longer need to support the `queries,scheduled_queries` queues?

Reply:
We did end up needing it - I misinterpreted the upgrade instructions. Re-added.